src/LogicApp/LogicApp/Utilities/LogicAppClientRunOperations.cs

```diff
@@ -40,9 +40,11 @@ public partial class LogicAppClient
         /// <param name="resourceGroupName">Name of the resource group</param>
         /// <param name="workflowName">Name of the workflow</param>
         /// <returns>List of workflow runs</returns>
-        public Page<WorkflowRun> GetWorkflowRuns(string resourceGroupName, string workflowName)
+        public Page<WorkflowRun> GetWorkflowRuns(string resourceGroupName, string workflowName, string nextPageLink = "")
```

Comment:
nit: please use constant string.Empty

Reply:
Didn't think that was possible here.

Reply:
Ah yes I forgot, it's not defined as a const but as a read-only static so we can't use it for default parameters.

Reply:
Not a fan of empty quotes either